tools/src/scie_pants/install_pants.py

```diff
@@ -78,22 +79,30 @@ def install_pants_from_pex(
     ptex: Ptex,
     extra_requirements: Iterable[str],
     find_links: str | None,
+    bootstrap_urls_path: str | None,
 ) -> None:
     """Installs Pants into the venv using the platform-specific pre-built PEX."""
     uname = os.uname()
-    major, minor = sys.version_info[:2]
-    pex_name = (
-        f"pants.{version}-cp{major}{minor}-{uname.sysname.lower()}_{uname.machine.lower()}.pex"
-    )
+    pex_name = f"pants.{version}-cp39-{uname.sysname.lower()}_{uname.machine.lower()}.pex"
+
     pex_url = f"https://github.com/pantsbuild/pants/releases/download/release_{version}/{pex_name}"
+    if bootstrap_urls_path:
+        bootstrap_urls = json.loads(Path(bootstrap_urls_path).read_text())
+        urls_info = bootstrap_urls["ptex"]
+        pex_url = urls_info.get(pex_name)
+        if pex_url is None:
+            raise ValueError(
+                f"Couldn't find '{pex_name}' in '{bootstrap_urls_path}' under the 'ptex' key."
```

Comment:
I personally find it helpful for tools to explain where they got values like this from, because it might be a random filename that's hard to grep for (as in, one might be confused about why scie-pants is looking in a particular file, having forgotten what's set in the environment). Hence:

```suggestion
                f"Couldn't find '{pex_name}' in PANTS_BOOTSTRAP_URLS file '{bootstrap_urls_path}' under the 'ptex' key."
```